src/ODETools/AffineThetaMethod.jl

```diff
@@ -0,0 +1,93 @@
+function solve_step!(uf::AbstractVector,
+                     solver::ThetaMethod,
+                     op::AffineODEOperator,
+                     u0::AbstractVector,
+                     t0::Real,
+                     cache) # -> (uF,tF)
+
+  dt = solver.dt
+  solver.θ == 0.0 ? dtθ = dt : dtθ = dt*solver.θ
+  tθ = t0+dtθ
+
+  if cache === nothing
+    ode_cache = allocate_cache(op)
+    l_cache = nothing
+  else
+    ode_cache, l_cache = cache
+  end
+
+  ode_cache = update_cache!(ode_cache,op,tθ)
+
+  afop = ThetaMethodAffineOperator(op,tθ,dtθ,u0,ode_cache)
```

Comment:
Idem as above, but this is veeery minor, since I have improved the implementation and the residual is computed now only once in the whole simulation...